ml_peg/models/models.py

```diff
@@ -17,6 +19,42 @@
 current_models = None
 
 
+def load_model_configs(
```

Comment:
Sorry I probably said to put it here, can it be moved to get_models.py? This module only really defines model classes, whereas that actually loads them in different was, as does this